config-templates/ibm/ibm-object-storage-plugin/2.2/custom-parameters.json

```diff
@@ -33,13 +33,24 @@
         "displayname": "COS Endpoint",
         "name": "cos-endpoint",
         "category": "config",
-        "required": "true"
+        "required": "false"
     },
     {
-        "description": "Enter COS storage class. For more info, refer to https://ibm.biz/cos-storage-classes",
+        "description": "Enter COS storage class. With cos-storageclass, provide value for either cos-endpoint or s3Provider, precedence is given to storageClass and s3Provider. To input endpoint explicitly, input only endpoint and storageClass and skip s3Provider. For more info, refer to https://ibm.biz/cos-storage-classes",
         "displayname": "COS storage class",
         "name": "cos-storageclass",
         "category": "config",
         "required": "true"
+    },
+    {
+        "description": "Enter COS s3Provider. Choose s3Provider as 'aws', 'wasabi' or 'ibm'",
```

Comment:
Enter the object storage service provider, supported providers are 'aws', 'ibm' and 'wasabi'"